app/_includes/prereqs/ollama.md

```diff
@@ -0,0 +1,32 @@
+To complete this tutorial, make sure you have Ollama installed and running locally.
+
+1. Visit the [Ollama download page](https://ollama.com/download) and download the installer for your operating system. Follow the installation instructions for your platform.
+
+2. After installation, open your terminal and run:
```

Comment:
````suggestion
1. Start Ollama:
   ```sh
   ollama start
   ```

2. After installation, open a new terminal window and run:
````
I had to start it before that model command would work. I installed via homebrew.